src/containers/Layout/ClaimRewardButton/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -42,6 +42,59 @@ exports[`ClaimRewardButton > it claims only selected rewards on submit button cl
 }
 `;
 
+exports[`ClaimRewardButton > it claims reward on submit button click and closes modal on success 1`] = `
+{
+  "accountAddress": "0x3d759121234cd36F8124C21aFe1c6852d2bEd848",
+  "claims": [
+    {
+      "contract": "legacyPoolComptroller",
+      "vTokenAddressesWithPendingReward": [
+        "0xD5C4C2e2facBEB59D0216D0595d63FcDc6F9A1a7",
+        "0xb7526572FFE56AB9D7489838Bf2E18e3323b441A",
+        "0x08e0A5575De71037aE36AbfAfb516595fE68e5e4",
+      ],
+    },
+    {
+      "comptrollerContractAddress": "0x10b57706ad2345e590c2ea4dc02faef0d9f5b08b",
+      "contract": "rewardsDistributor",
+      "contractAddress": "0xa14c236372228b6e8182748f3ebbfb4bfeea3574",
+      "vTokenAddressesWithPendingReward": [
+        "0xcfc8a73f9c888eea9af9ccca24646e84a915510b",
+        "0x12d3a3aa7f4917ea3b8ee34f99a9a7eec521fa61",
+      ],
+    },
+    {
+      "comptrollerContractAddress": "0x10b57706ad2345e590c2ea4dc02faef0d9f5b08b",
+      "contract": "rewardsDistributor",
+      "contractAddress": "0xadbed07126b7b70cbc5e07bf73599d55be571b9c",
+      "vTokenAddressesWithPendingReward": [
+        "0xcfc8a73f9c888eea9af9ccca24646e84a915510b",
+      ],
+    },
+    {
+      "contract": "vaiVault",
+    },
+    {
+      "contract": "xvsVestingVault",
+      "poolIndex": 0,
+      "rewardToken": {
+        "address": "0xB9e0E753630434d7863528cc73CB7AC638a7c8ff",
+        "asset": "/src/packages/tokens/img/xvs.svg",
+        "decimals": 18,
+        "symbol": "XVS",
+      },
+    },
+    {
+      "contract": "prime",
+      "vTokenAddressesWithPendingReward": [
+        "0xcfc8a73f9c888eea9af9ccca24646e84a915510b",
+        "0x12d3a3aa7f4917ea3b8ee34f99a9a7eec521fa61",
+      ],
+    },
+  ],
+}
+`;
+
 exports[`ClaimRewardButton > it claims reward on submit button click on success 1`] = `
```

Comment:
It looks like the snapshot for the test that was renamed is still here, when it should have been removed/replaced

Reply:
Good catch